editor.planx.uk/src/pages/Preview/Node.tsx

```diff
@@ -169,7 +170,7 @@ const Node: React.FC<any> = (props: Props) => {
       return <FindProperty {...allProps} />;
 
     case TYPES.List:
-      return <List {...allProps} />;
+      return hasFeatureFlag("LIST_COMPONENT") ? <List {...allProps} /> : null;
```

Comment:
Moving this here means I can escape the "don't use conditional hooks" warning.

Docs: https://react.dev/reference/rules/rules-of-hooks#only-call-hooks-at-the-top-level